### DISCUSSION TITLE
How to use multiple ssh public keys

### DISCUSSION THREAD

**Discussion by sylvorg** [FiloSottile/age#525] [Q&A] [answered]
Hello!

So [in the documentation](https://github.com/FiloSottile/age#ssh-keys), it is stated that you can use the `-R` option to specify an ssh public key to encrypt a file to. It was also stated previously that `-R` is used to pass in a recipients file. Does this mean that I have to specify `-R` multiple times to encrypt a file to multiple ssh keys, or is there a way to add all the ssh public keys to one file and use that instead, similar to how the age public keys can be grouped?

Thank you kindly for the help!

**Comment by FiloSottile** [comment, marked as answer]
You can have multiple ssh keys in a file, one per line!

**Reply by sylvorg** [reply]
So a `recipients.txt` file or otherwise with a bunch of ssh public keys (one per line)?

**Reply by FiloSottile** [reply]
Yep! They work pretty much the same as age keys.

**Reply by sylvorg** [reply]
Got it; thanks for the help!
